djongo/models/fields.py

```diff
@@ -117,14 +117,14 @@ class DictField(Field):
     def get_db_prep_value(self, value, connection, prepared=False):
         if not isinstance(value, dict):
             raise ValueError(
-                f'Value: {value} must be of type dict'
+                f'Value: {value} must be of type list'

```

Comment:
```suggestion
                f'Value: {value} must be of type dict'
```
Why exactly has this been changed?